src/utils/web3-utils.js

```diff
@@ -1,14 +1,27 @@
 import { ethers, providers as Providers } from 'ethers'
 import { getPreferredChain } from '../local-settings'
+import {
+  ARBISCAN_NETWORK_TYPES,
+  ARBISCAN_TYPES,
+  BLOCKSCOUT_NETWORK_TYPES,
+  BLOCKSCOUT_TYPES,
+  ETHERSCAN_NETWORK_TYPES,
+  ETHERSCAN_TYPES,
+  POLYGONSCAN_NETWORK_TYPES,
+  POLYGONSCAN_TYPES,
+} from './provider-types'
 
 const DEFAULT_LOCAL_CHAIN = ''
+
 const networks = {
```

Comment:
Let's add mainnet as well:

```suggestion
const networks = {
mainnet: {
    chainId: 1,
    ensRegistry: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
    name: 'Mainnet',
    type: 'mainnet',
    defaultEthNode: 'https://eth-rpc.gateway.pokt.network',
    explorer: 'etherscan',
    nativeToken: 'ETH',
  },
```

Check that the ens registry is correct.